src/message.rs

```diff
@@ -1876,6 +1912,36 @@ mod tests {
         assert_eq!(url, "https://j.si/foo");
     }
 
+    #[async_std::test]
+    async fn test_create_webrtc_instance() {
+        let instance = Message::create_webrtc_instance("https://meet.jit.si/", "123");
+        assert_eq!(instance, "https://meet.jit.si/123");
+
+        let instance = Message::create_webrtc_instance("https://meet.jit.si", "456");
+        assert_eq!(instance, "https://meet.jit.si/456");
+
+        let instance = Message::create_webrtc_instance("meet.jit.si", "789");
+        assert_eq!(instance, "https://meet.jit.si/789");
+
+        let instance = Message::create_webrtc_instance("bla.foo?", "123");
+        assert_eq!(instance, "https://bla.foo?123");
+
+        let instance = Message::create_webrtc_instance("jitsi:bla.foo#", "456");
+        assert_eq!(instance, "jitsi:https://bla.foo#456");
+
+        let instance = Message::create_webrtc_instance("bla.foo#room=", "789");
+        assert_eq!(instance, "https://bla.foo#room=789");
+
+        let instance = Message::create_webrtc_instance("http://bla.foo#room", "123");
+        assert_eq!(instance, "http://bla.foo#room/123");
```

Comment:
@r10s Regarding insecure HTTP, I found this: https://blog.mozilla.org/webrtc/camera-microphone-require-https-in-firefox-68/ So, I think we should not try to "offer support" for it either.

Reply:
well, video may not only go through browsers, this can also be used to route through some app, maybe just in the intranet, whatever. so, in general, i would not say we "offer support" for something at all here. but i agree, the test looks like a typo, i will change this.